libs/core/web/src/lib/layout/menu/MenuApp.tsx

```diff
@@ -65,6 +90,12 @@ export function MenuApp() {
                 {t(i18nKeys.lang.english)}
               </MenuItem>
             </Menu>
+
+            {isLogged && (
+              <IconButtonStyled func={logout}>
```

Comment:
I usually avoid naming components with `Styled`, name it IconButton directly, or give it a name that indicates what's special about it.